Doc/library/configparser.rst

```diff
@@ -136,6 +136,26 @@ sections [1]_.  Note also that keys in sections are
 case-insensitive and stored in lowercase [1]_.
 
```

Comment:
nit: single line here.

Reply:
This line is not modified by this PR. So I wonder if it's a good place to fix it.

What do you think about it?

Reply:
What I know is there should be a single line before the next paragraph in a section.

Reply:
I think it's ok now.